backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -0,0 +1,35 @@
+[id="about-dr"]
+= About disaster recovery
+include::modules/common-attributes.adoc[]
+:context: about-dr
+
+toc::[]
+
+The disaster recovery documentation provides information for administrators on
+how to recover from several disaster situations that might occur with their
+{product-title} cluster. As an administrator, you might need to follow one or
+more of the following procedures in order to return your cluster to a working
+state.
+
+xref:../../backup_and_restore/disaster_recovery/scenario-1-infra-recovery.adoc#dr-infrastructure-recovery[Recovering from lost master hosts]::
+This solution handles situations where you have lost the majority of your master
+hosts, leading to etcd quorum loss and the cluster going offline. As long as you
+have taken an etcd backup and have at least one remaining healthy master host,
+you can follow this procedure to recover your cluster.
```

Comment:
if you are rolling back time (using etcd backup) you need to recover static pod manifests and data as well

Reply:
There should be a section on how to backup them as well linked from the etcd. you should always backup both

Reply:
> if you are rolling back time (using etcd backup) you need to recover static pod manifests and data as well

@tnozicka this makes sense. So are all static pods managed by machine config? I know etcd is but it does not change. If not how do we restore them to previous state? If yes is there a mechanism to do so currently in DR?

Reply:
Static pods are not managed by MCO but by it's operator which installs them to a node. They are part of the cluster state just as etcd is. Restore and backup would be `tar` or similar.

Reply:
so to confirm backing up and restore would involve matching of both assets.

Reply:
this makes sense cc @bergerhoffer